hr_attendance_calendar/wizards/hr_attendance_settings.py

```diff
@@ -0,0 +1,46 @@
+# -*- coding: utf-8 -*-
+##############################################################################
+#
+#    Copyright (C) 2018 Compassion CH (http://www.compassion.ch)
+#    @author: Stephane Eicher <seicher@compassion.ch>
+#
+#    The licence is in the file __manifest__.py
+#
+##############################################################################
+
+from odoo import models, fields, api
+
+
+class HrAttendanceSettings(models.TransientModel):
+    """ Settings configuration for hr.attendance."""
+    # _name = "hr.attendance.settings"
```

Comment:
So you are just extending the base settings? Then you should remove all the views you did for `hr.attendance.settings` or it will fail to install. Or maybe just uncomment this?